pom.xml

```diff
@@ -151,6 +151,26 @@
                         </headerDefinitions>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>xml-maven-plugin</artifactId>
+                    <version>${xml-maven-plugin.version}</version>
+                    <configuration>
+                        <indentSize>2</indentSize>
+                        <formatFileSets>
+                            <formatFileSet>
+                                <directory>${project.basedir}</directory>
+                                <includes>
+                                    <include>**/*.xml</include>
+                                </includes>
+                                <excludes>
+                                    <exclude>**/.*</exclude>
+                                </excludes>
+                            </formatFileSet>
```

Comment:
I excluded  target  directory not including `<exclude>`  but excluding `<include>` at  af632050d11b237e04361d2073d1a10ad755f03c commit .